app/presenters/pafs_core/validation_presenter.rb

```diff
@@ -310,5 +311,19 @@ def risks_error
                 "and the households benefiting.")
       false
     end
+
+    private
+
+    def funding_calculator_correct_version?
```

Comment:
Why is the calculator version validation logic split across `ValidationPresenter` and `FundingCalculatorVersion` ?

Reply:
Prior to this change, the calculator version was only checked on upload and not on submission. The only check before submission was that there was a file that was uploaded, under the assumption that all of the validation for that file had been done on upload. The ValidationPresenter is only used when a project is being submitted, and each step completes its own validation for what is being submitted for that step. I thought it was best to leverage the existing code used for the validation on the step. Essentially, to get to the point of submitting a project, the user would have to have uploaded a version of the PF calculator that was valid at the time. This extra validation on submission is to ensure that it is still valid. If, in the future, the version of the calculator is updated again, a change should only be needed in one place.